src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -948,11 +955,22 @@ void IgnRenderer::Render(RenderSync *_renderSync)
       {
         root.Load(this->dataPtr->spawnSdfPath);
       }
+      else if (!this->dataPtr->spawnCloneName.empty())
```

Comment:
Could you also add this to the new `Spawn` plugin from https://github.com/ignitionrobotics/ign-gazebo/pull/983?